packages/react/src/NavList/NavList.tsx

```diff
@@ -288,6 +290,54 @@ const Group: React.FC<NavListGroupProps> = ({title, children, sx: sxProp = defau
   )
 }
 
+export type NavListShowMoreItemProps = {
+  children: React.ReactNode
+  label: string
```

Comment:
Could this have a default value of "Show more" or is there really not a good default for the label that is used most of the time?

Reply:
We definitely could! I feel like most use cases will have context for what the items in the `NavList` are for. So adding a default "Show more" shouldn't hurt, as we allow consumers to customize it further if more context is needed.